### PR TITLE
adding the postgres scripts to the new repo

### PR DESCRIPTION
**Pull request by ecstaticrainbow**
Co-authored-by: RD <47347806+WorkRD@users.noreply.github.com>
Co-authored-by: JZ <32240008+jarekzaj@users.noreply.github.com>

### CODE REVIEW

File: Postgres/scaleDownPostgres.ps1 [hunk 1]
```diff
@@ -0,0 +1,60 @@
+$resourceURI = Get-AutomationVariable -Name 'PostgresResourceURI'
+$tenantId = Get-AutomationVariable -Name 'TennantID'
+$clientId =  Get-AutomationVariable -Name 'ClientId'
+$clientSecret =  Get-AutomationVariable -Name 'ClientSecret'
+
+$UnsecurePassword = $clientSecret
```
Comment (WorkRD) [inline Postgres/scaleDownPostgres.ps1]: Can we remove this and just use $clientSecret?

File: Postgres/scaleUpPostgres.ps1 [hunk 2]
```diff
@@ -0,0 +1,60 @@
+$resourceURI = Get-AutomationVariable -Name 'PostgresResourceURI'
+$tenantId = Get-AutomationVariable -Name 'TennantID'
+$clientId =  Get-AutomationVariable -Name 'ClientId'
+$clientSecret =  Get-AutomationVariable -Name 'ClientSecret'
+
+$UnsecurePassword = $clientSecret
```
Comment (WorkRD) [inline Postgres/scaleUpPostgres.ps1]: Same as before- can we just use $clientSecret

**Review by WorkRD — CHANGES_REQUESTED** [review]

File: Postgres/scaleDownPostgres.ps1 (hunk 1 above)
Comment (ecstaticrainbow) [reply Postgres/scaleDownPostgres.ps1]:
```suggestion

```

File: Postgres/scaleUpPostgres.ps1 (hunk 2 above)
Comment (ecstaticrainbow) [reply Postgres/scaleUpPostgres.ps1]:
```suggestion
```

**Review by WorkRD — APPROVED** [review]

**Pull request merged**